packages/@lwc/integration-karma/test-hydration/directives/lwc-is/index.spec.js

```diff
@@ -0,0 +1,22 @@
+export default {
+    props: {
+        label: 'dynamic',
+    },
```

Comment:
What happens if the server ctor is different from the client ctor? We would get a hydration warning/error in that case, correct? Can we add a test for that?

Reply:
I added a new [test](https://github.com/salesforce/lwc/blob/1658d36d1a001bfce3a4a441323e9d4caf473f48/packages/%40lwc/integration-karma/test-hydration/mismatches/dynamic-component/index.spec.js) for this scenario.